packages/server/graphql/mutations/startRetrospective.ts

```diff
@@ -103,8 +103,7 @@ export default {
     }
 
     const updates = {
-      lastMeetingType: meetingType,
-      updatedAt: new Date()
```

Comment:
We can drop this because:
1. We don't write to RethinkDB anymore
2. We have the [trigger function](https://github.com/ParabolInc/parabol/blob/1a2e266d02a97bc1f6042fe399f82570636d35dc/packages/server/postgres/migrations/1652361575188_addUpdatedAtTrigger.ts#L9) defined in the Team table in PG.